Kibana stats integration test failling on version key not found.

found in https://travis-ci.org/elastic/beats/jobs/506466210

```
ok  	github.com/elastic/beats/metricbeat/module/kvm/dommemstat	0.007s	coverage: 48.1% of statements
command [go test -tags integration -cover -coverprofile /tmp/gotestcover-932530723 github.com/elastic/beats/metricbeat/module/kibana/stats]: exit status 1
Killing metricbeat43bb03fcf1fb8ccdc91622c74e2e2b37303cbde9_etcd_1 ... 
	stats_integration_test.go:60: 
			Error Trace:	stats_integration_test.go:60
			Error:      	Should be empty, but was [HTTP error 500 in stats: 500 Internal Server Error]
			Test:       	TestFetch
	stats_integration_test.go:61: 
			Error Trace:	stats_integration_test.go:61
			Error:      	Should NOT be empty, but was []
			Test:       	TestFetch
--- FAIL: TestData (1.03s)
	stats_integration_test.go:76: getting kibana version key not found
FAIL
coverage: 13.3% of statements
FAIL	github.com/elastic/beats/metricbeat/module/kibana/stats	149.446s
```

--- a/metricbeat/module/kibana/_meta/Dockerfile
+++ b/metricbeat/module/kibana/_meta/Dockerfile
@@ -1,2 +1,3 @@
 FROM docker.elastic.co/kibana/kibana:6.6.0
-HEALTHCHECK --interval=1s --retries=300 CMD curl -f http://localhost:5601/api/status | grep '"disconnects"'
+HEALTHCHECK --interval=1s --retries=300 --start-period=60s CMD python -c 'import urllib, json; response = urllib.urlopen("http://myelastic:changeme@localhost:5601/api/status"); data = json.loads(response.read()); exit(1) if data["status"]["overall"]["state"] != "green" else exit(0);'
+

--- a/metricbeat/module/kibana/stats/stats_integration_test.go
+++ b/metricbeat/module/kibana/stats/stats_integration_test.go
@@ -36,7 +36,6 @@ import (
 )
 
 func TestFetch(t *testing.T) {
-	t.Skip("flaky")
 	compose.EnsureUpWithTimeout(t, 600, "elasticsearch", "kibana")
 
 	config := mtest.GetConfig("stats")
@@ -68,7 +67,6 @@ func TestFetch(t *testing.T) {
 }
 
 func TestData(t *testing.T) {
-	t.Skip("flaky")
 	compose.EnsureUp(t, "kibana")
 
 	config := mtest.GetConfig("stats")

